### PR TITLE
Update contentDigest wording to clarify behavior

### PR DESCRIPTION
**Pull request by jeremyrickard**
This PR tries to capture the discussion regarding digest validation present in issue #254 and discussions that occurred at the last CNAB community meeting. 

Fixes: #254

### CODE REVIEW

File: 101-bundle-json.md [hunk 1]
```diff
@@ -313,7 +313,7 @@ The `imageType` field MUST describe the format of the image. The list of formats
...
+The `contentDigest` field MUST contain a digest, in [OCI format](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests), to be used to compute the integrity of the image. The calculation of how the image matches the contentDigest is dependent upon image type. (OCI, for example, uses a Merkle tree while VM images are checksums). During bundle development, it may be ideal to omit the contentDigest field and/or skip validation. Once a bundle is ready to be transmitted as a thick or thin bundle, it must have a contentDigest field. If a contentDigest field is present, a runtime MUST validate the image diget prior to executing an action. If the contentDigest is not present, the runtime SHOULD report an error so the user is aware that there is no contentDigest provided. Runtimes MAY allow users to override this behavior and perform actions on bundles that do NOT have contentDigest values populated.
```
Comment (radu-matei) [inline 101-bundle-json.md]: nit: formatting for `contentDigest`

File: 101-bundle-json.md [hunk 2]
```diff
@@ -313,7 +313,7 @@ The `imageType` field MUST describe the format of the image. The list of formats
...
+The `contentDigest` field MUST contain a digest, in [OCI format](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests), to be used to compute the integrity of the image. The calculation of how the image matches the `contentDigest` is dependent upon image type. (OCI, for example, uses a Merkle tree while VM images are checksums). During bundle development, it may be ideal to omit the `contentDigest` field and/or skip validation. Once a bundle is ready to be transmitted as a thick or thin bundle, it must have a `contentDigest` field. If a `contentDigest` field is present, a runtime MUST validate the image diget prior to executing an action. If the `contentDigest` is not present, the runtime SHOULD report an error so the user is aware that there is no `contentDigest` provided. Runtimes MAY allow users to override this behavior and perform actions on bundles that do NOT have `contentDigest` values populated.
```
Comment (youreddy) [inline 101-bundle-json.md]: typo in the word digest at "runtime MUST validate the image diget prior to"

Comment (youreddy) [reply 101-bundle-json.md]: Also, I don't think "NOT" needs to be capitalized in the last line.

Comment (jeremyrickard) [reply 101-bundle-json.md]: Thanks! Got them both.

**Comment by jeremyrickard** [comment]
@youreddy take another look please!

**Comment by jeremyrickard** [comment]
Thanks @technosophos. If there is anything else @youreddy, LMK tomorrow and I'll do another PR.

**Comment by youreddy** [comment]
Everything looks good to me! Thanks @jeremyrickard

**Pull request merged**